src/persistentmodel/persistentstore.h

```diff
@@ -7,34 +7,51 @@
 
 #define DEFAULT_DBNAME "tarsnap.db"
 
+/*!
+ * \ingroup persistent
+ * \brief The PersistentStore is a singleton which interfaces with a database
+ * which stores lists of archives, jobs, and journal entries.
+ */
 class PersistentStore : public QObject
 {
     Q_OBJECT
 
 public:
     ~PersistentStore();
+    //! Singleton.
     static PersistentStore &instance()
     {
         static PersistentStore instance;
         if(!instance.initialized())
             instance.init();
         return instance;
     }
+    //! Locks the database, upgrades the version if it is old, creates a new
```

Comment:
My apologies; I misread `initialized()` for `init()`.  I'm creating a fixed version now.